test/index.test.ts

```diff
@@ -302,6 +302,27 @@ test('ensure run and exec with command defined as array are working', async ():
   await compose.down({ cwd: path.join(__dirname), log: logOutput });
 });
 
+test('build accepts config as string', async (): Promise<void> => {
+  const configuration = await new Promise(function (resolve, reject) {
+    readFile(path.join(__dirname, 'docker-compose-2.yml'), function (err, content) {
+      if (err) {
+        return reject(err);
+      }
+      return resolve(content.toString());
+    })
+  });
```

Comment:
Promise resolution function does not actually return anything (at least it shouldn't).
```suggestion
  const configuration = await new Promise(function (resolve, reject): void {
    readFile(path.join(__dirname, 'docker-compose-2.yml'), function (err, content) {
      if (err) {
        reject(err);
      }
      resolve(content.toString());
    })
  });
```